doc/radosgw/account.rst

```diff
@@ -187,6 +187,43 @@ Alternatively, you may want to create a new account for each existing user. In
 that case, you may want to add the ``--account-root`` option to make each user
 the root user of their account.
 
+Migrating Notification Topics
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Account topics are only supported when ``notification_v2`` feature is enabled,
+as described in `Bucket Notifications`_ and `Supported Zone Features`_.
+
+1. ``Migration Impact``: When a non-account user is migrated to an account,
+the user loses access to its existing notification topics via the SNS Topic API.
+However, ongoing notifications remain unaffected.
```

Comment:
I'm unclear what "ongoing" here means vs existing notification topics, to someone not versed in this area this sounds like a contradiction.

Reply:
maybe "However, bucket notifications on these topics will continue to be delivered."

Reply:
> maybe "However, bucket notifications on these topics will continue to be delivered."

Just saw this after adding the new evidence but the latest change is on the similar lines:

> However, these topics remain functional, and bucket notifications will continue
to be delivered as expected.

Let me know how it looks.

Reply:
even better